WTWSMS/common/minor_titles/WtWSMS_minor_titles.txt

```diff
@@ -132,32 +132,37 @@ title_sigecempa = {
 }
 
 title_court_celtic_poet = { # Celtic Poet/Bard
-	dignity = 0.25
+	dignity = 1
 	realm_in_name = yes
 	grant_limit = 1
 	opinion_effect = 30
 	
 	monthly_salary = 0.02
-	monthly_prestige = 0.01
+	monthly_prestige = 0.1
 	
 	allowed_to_hold = {
 		is_adult = yes
 		is_ruler = no
 		is_female = no
-		trait = poet
+		OR = {
+			trait = poet
+			learning = 14
+		}
```

Comment:
> Anyone of sufficient literacy and imagination can make poems. In the case of this game, fourteen IQ is very high and demonstrates an interest in intellectual pursuits. This should stay so that the title isn't eternally unused.

It isn't a poetry school here, it is about patronising already existant poets. There are already events for creating new ones in vanilla, and otherwise we can add a decision. I don't get why you insist on this when I told you your change doesn't fit in. 

> However, I am open to an Event where the Court Poet, with certain Traits, doesn't want to make poetry and submits his resignation for your request. That would remove his Poet Trait as well.

You do realise this is the exact opposite of what I said? 😛 Did you misunderstand or is this on purpose?